tests/network/vmi_multus.go

```diff
@@ -1041,7 +1042,19 @@ var _ = Describe("[Serial]SRIOV", func() {
 				migrationUID := tests.RunMigrationAndExpectCompletion(virtClient, migration, tests.MigrationWaitTime)
 				tests.ConfirmVMIPostMigration(virtClient, vmi, migrationUID)
 
-				Expect(checkMacAddress(vmi, interfaceName, mac)).To(Succeed(),
+				var interfaceName string
+
+				// It may take some time for the VMI interface status to be updated with the information reported by
+				// the guest-agent.
+				Eventually(func() error {
+					var err error
+					vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+					Expect(err).NotTo(HaveOccurred())
+					if interfaceName, err = getInterfaceNameByMAC(vmi, mac); err != nil {
```

Comment:
AFAIU, you're not using this `interfaceName` variable outside this scope.

Why does it need to be a variable scoped to the test's body ?

Reply:
Done